install_config/registry/extended_registry_configuration.adoc

```diff
@@ -82,40 +82,40 @@ unnecessary downtime for pods and does not actually clear the cache.
 [[whitelisting-docker-registries]]
 == Whitelisting Docker Registries
 
-You can specify a whitelist of docker registries, allowing you to curate a set
-of images and templates that are available for download by {product-title}
-users. This curated set can be placed in one or more docker registries, and then
-added to the whitelist. When using a whitelist, only the specified registries
-are accessible within {product-title}, and all other registries are denied
-access by default.
+You can specify a whitelist of Docker registries, allowing you to curate a set 
```

Comment:
The registries configuration isn't a whitelist.  A client can pull from any registry that uses SSL/TLS (i.e., a "secure" registry) as long as it's not in the `registries.block` list (i.e., "blacklist").  The registries configuration only exists to allow customization of the search, insecure, and block list.